### ISSUE TITLE
Hyprland crashes when draging window to other workspace

### ISSUE THREAD

**Issue by Postboote1** [KZDKM/Hyprspace#84]
Hello
I recently installed hyprspace on my nixos Laptop.
But the problem is, that when I open overview and drag a window from the overview to another workspace hyprland crashes and it takes me back to sddm. When launching hyprland in tty and doing the same it also crashes but wirh no crash report.

I am using Nixos with flakes: NixOS 24.11.20240828.71e91c4 
Kernel: Linux 6.10.6

**Comment by myamusashi** [comment]
Which hyprland version are you using? I have the same problem with Hyprland-git on arch, and I tested on Ubuntu with hyprland 0.42.0 and no problem with that, maybe something wrong with the Hyprland but idk tho

**Comment by Postboote1** [comment]
Since I use nixos with hyperland as flake input from git I think it is the newest

**Comment by Postboote1** [comment]
Since I use nixos with hyperland as flake input from git I think it is the newest

**Comment by myamusashi** [comment]
Yeah, something wrong or change in Hyprland, i'm using the 0.42.0 no problem with the draging window

**Comment by KZDKM** [comment]
Cant reproduce on 0.42 either. Git changes are not tracked and might break the plugin, I recommend you to use tagged 0.42 version if there are no substantial reason for you to stay on git.

**Comment by Postboote1** [comment]
I am using nixos. So using Git is the only option the keep up and use submodules/plugins correct
